src/main/java/task/second/CallableImpl.java

```diff
@@ -0,0 +1,25 @@
+package task.second;
+
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class CallableImpl implements Callable<Integer> {
```

Comment:
let's name class according to its' purpose

Reply:
done